frontend/app/src/router.tsx

```diff
@@ -145,6 +145,32 @@ export const router = createBrowserRouter([
               },
             ],
           },
+          {
+            path: "role-management",
+            lazy: () => import("@/pages/role-management"),
+            children: [
+              {
+                index: true,
+                lazy: () => import("@/screens/role-management/accounts"),
+              },
+              {
+                path: "groups",
+                lazy: () => import("@/screens/role-management/groups"),
+              },
+              {
+                path: "roles",
+                lazy: () => import("@/screens/role-management/roles"),
+              },
+              {
+                path: "global-permissions",
+                lazy: () => import("@/screens/role-management/global-permissions"),
+              },
+              {
+                path: "object-permissions",
+                lazy: () => import("@/screens/role-management/object-permissions"),
```

Comment:
we should move these files to `/pages`

Reply:
why? it's the same for IPAM, and the content is part of a page since the page itself is the navigation + outlet